requirements_dev.txt

```diff
@@ -3,4 +3,5 @@ markdown
 lxml
 pyenchant
 ipython
+matplotlib
```

Comment:
No need for matplotlib in the requirements_dev.txt as it is included through the requirements.txt.